cmgd/cmgd_bcknd_adapter.c

```diff
@@ -841,6 +841,7 @@ static int cmgd_bcknd_adapter_conn_init(struct thread *thread)
 
 	adptr = (cmgd_bcknd_client_adapter_t *)THREAD_ARG(thread);
 	assert(adptr && adptr->conn_fd);
+	adptr->conn_init_ev = NULL;
```

Comment:
This could be a memory leak. Need discuss more. Doesn't seem to be a proper fix.

Reply:
Once we schedule the thread we need to set the thread variable to NULL. Else it will get rescheduled.

Reply:
No. That should not be the case. Once the event handler is scheduled it does not matter if adapter->conn_init_ev is NULL or not.